modules/market/src/main/java/com/opengamma/strata/market/curve/CurveCalibrationInfo.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.market.curve;
+
+/**
+ * Information about curve calibration.
+ * <p>
+ * Implementations of this marker interface provide information that was
+ * obtained during curve calibration, such as a Jacobian.
+ */
+public interface CurveCalibrationInfo {
```

Comment:
Why do we need this interface? Why not have:

    CurveInfoType<Jacobean> JACOBEAN = CurveInfoType.of("Jacobean");

What do we gain with the extra level of indirection where we have to cast `CurveCalibrationInfo` to `JacobeanCurveCalibrationInfo`?